dubbo-spring-boot-autoconfigure/src/main/java/org/apache/dubbo/spring/boot/autoconfigure/DubboRelaxedBinding2AutoConfiguration.java

```diff
@@ -32,16 +33,13 @@
 import org.springframework.core.env.MutablePropertySources;
 import org.springframework.core.env.PropertyResolver;
 
+import java.util.LinkedHashSet;
 import java.util.Map;
 import java.util.Set;
 
 import static com.alibaba.spring.util.PropertySourcesUtils.getSubProperties;
 import static java.util.Collections.emptySet;
-import static org.apache.dubbo.spring.boot.util.DubboUtils.BASE_PACKAGES_BEAN_NAME;
-import static org.apache.dubbo.spring.boot.util.DubboUtils.BASE_PACKAGES_PROPERTY_NAME;
-import static org.apache.dubbo.spring.boot.util.DubboUtils.DUBBO_PREFIX;
-import static org.apache.dubbo.spring.boot.util.DubboUtils.DUBBO_SCAN_PREFIX;
-import static org.apache.dubbo.spring.boot.util.DubboUtils.RELAXED_DUBBO_CONFIG_BINDER_BEAN_NAME;
+import static org.apache.dubbo.spring.boot.util.DubboUtils.*;
```

Comment:
@titi1231 pls install [codestyle check rule](https://github.com/apache/dubbo/blob/master/codestyle/dubbo_codestyle_for_idea.xml) to your IDE

wildcard import is not allowed in Dubbo Repo